android/app/src/main/res/layout/fragment_editor.xml

```diff
@@ -214,6 +214,10 @@
           android:id="@+id/block_website"
           layout="@layout/item_editor_input"/>
 
+        <include
+            android:id="@+id/block_website_menu"
+            layout="@layout/item_editor_input"/>
```

Comment:
IMO it'd be better to move this below the Wi-Fi section (`id/block_wifi`), as adding wifi is more useful and likely to do when surveying

Reply:
Yeah, but in the case of the website:menu is a “sub-tag” of website, I think that putting them in a row would be more coherent than separating them by the wifi tag.

Reply:
mm fair enough, seeing it in the screenshot it doesn't look too bad